src/tools/profile/read_profile.c

```diff
@@ -481,12 +494,17 @@ int parse_args(int argc, char **argv, options_t *opts)
             }
             break;
         case 'h':
+            usage();
+            return -127;
```

Comment:
Shouldn't `ucx_read_profile -h` return 0 since it successfully displays usage information to the user?

Reply:
this is the return value from the function, return value from main() would be 0